integration_tests/dbt_project.yml

```diff
@@ -2,18 +2,18 @@ name: 'integration_tests'
 version: '1.0.0'
 config-version: 2
 
-profile: default
```

Comment:
I don't think this change was intended. Same goes for changing the default for `use_syntethic_data` and `input_schema`.

Reply:
yup, thanks!  should be good now